drools-wb-webapp/pom.xml

```diff
@@ -1262,26 +1275,191 @@
       <artifactId>org.eclipse.sisu.inject</artifactId>
       <scope>runtime</scope>
     </dependency>
+
+    <!-- Included dependencies because previously where cross-provided by other components. -->
+    <dependency>
+      <groupId>org.uberfire</groupId>
+      <artifactId>uberfire-wires-core-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.uberfire</groupId>
+      <artifactId>uberfire-wires-core-grids</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.uberfire</groupId>
+      <artifactId>uberfire-wires-core-trees</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.errai</groupId>
+      <artifactId>errai-html5</artifactId>
+    </dependency>
+
+    <!-- Added because of dependency:analyze result -->
+    <dependency>
+      <groupId>org.uberfire</groupId>
+      <artifactId>uberfire-wires-core-client</artifactId>
+      <scope>compile</scope>
```

Comment:
Compile scope is the default scope. So if you don't declare scope for a dependency, it is automatically compile scoped (just so you know for next time and can save some boilerplate declarations in the future)